packages/calypso-e2e/src/lib/flows/gutenboarding-flow.ts

```diff
@@ -262,7 +262,10 @@ export class GutenboardingFlow {
 		// Clicking on a language button triggers a navigation to a URL containing
 		// the ISO 639-1 code eg. /new/ja.
```

Comment:
Small nit, but this comment is out of date now!

Reply:
Thanks, I've updated it!